src/cable/remote_connections.cr

```diff
@@ -0,0 +1,31 @@
+module Cable
+  class RemoteConnections
+    def initialize(@server : Cable::Server)
+    end
+
+    # Specify the name and value of your connection's `identified_by`
+    # ```
+    # # e.g.
+    # # identified_by :user_id
+    # # self.user_id = 1234.to_s
+    #
+    # where({:user_id, "1234"})
+    # ```
+    def where(identifier : Tuple(Symbol, String))
+      RemoteConnection.new(@server, identifier[1])
+    end
+
```

Comment:
This could be a `**identifiers` NamedTuple, but I couldn't think of a reason why you would pass more than 1 key... Technically, this would work

```crystal
identified_by :identifier1
identified_by :identifier2
identified_by :identifier3

def connect
  self.identifier1 = "1"
  self.identifier2 = "2"
  self.identifier3 = "3"
end
```

But why would you do that?

Also, the first item in the tuple isn't used, so we could remove it... but then you'll see something like this:

```
remote_connections.where("1234")
```

and at that point, why even call it `where`? :man_shrugging:  We could rename it to `remote_connections.with_identifier("1234")` :thinking: